clang/include/clang/Basic/DiagnosticParseKinds.td

```diff
@@ -413,6 +413,9 @@ def err_function_scope_depth_exceeded : Error<
   "function scope depth exceeded maximum of %0">, DefaultFatal;
 def err_missing_comma_before_ellipsis : Error<
   "C requires a comma prior to the ellipsis in a variadic function type">;
+def warn_deprecated_missing_comma_before_ellipsis : Warning<
+  "variadic parameters that are not preceded by a comma are deprecated">,
```

Comment:
Maybe "declaration of a variadic function without a comma before '...' is deprecated"

("variadic parameter" is not defined and somewhat confusing)

Reply:
I've updated the diagnostic to use your suggested wording.